spec/koans.js

```diff
@@ -85,54 +89,69 @@ describe('`string.includes()` finds string within another string. ', () => {
 
   describe('find a single character', function() {
     it('in a three char string', function() {
-      /* const searchString = ???? */
-      //expect('xyz'.includes(searchString)).toBe(true);
+      const searchString = "z";
+      expect('xyz'.includes(searchString)).toBe(true);
     });
     it('reports false if character was not found', function() {
-      /* const expected = ????*/;
-      //expect('xyz'.includes('abc')).toBe(expected);
+      const expected = false;
+      expect('xyz'.includes('abc')).toBe(expected);
     });
   });
 
   describe('find a string', function() {
     it('that matches exactly', function() {
       /* const findSome = .... => 'xyz'.includes();*/
-      //expect(findSome('xyz')).toBe(true);
+      // const findSome  = str => str.includes("xyz");
+      const findSome =
+      function (str){
+        return "xyz".includes(str)
+      }
+      expect(findSome('xyz')).toBe(true);
     });
   });
 
   describe('search for an empty string, is always true', function() {
     it('in an empty string', function() {
       /* .... */
-      //expect(''.includes(x)).toBe(true);
+      const x = "";
+      expect(''.includes(x)).toBe(true);
     });
     it('in `abc`', function() {
       /* .... */
-      //expect('abc'.includes(x)).toBe(true);
+      const x = "abc";
+      expect('abc'.includes(x)).toBe(true);
```

Comment:
Yep ! But the point of this test is to show that looking for an empty string in any kind of string will always return true, so you can also try with 
```js
 const x = "";
```